Mathlib/Data/Int/SuccPred.lean

```diff
@@ -54,24 +54,24 @@ theorem succ_iterate (a : ℤ) : ∀ n, (succ^[n]) a = a + n
   | 0 => (add_zero a).symm
   | n + 1 => by
     rw [Function.iterate_succ', Int.ofNat_succ, ← add_assoc]
-    exact congr_arg _ (succ_iterate n)
+    exact congr_arg _ (succ_iterate a n)
 #align int.succ_iterate Int.succ_iterate
 
 theorem pred_iterate (a : ℤ) : ∀ n, (pred^[n]) a = a - n
   | 0 => (sub_zero a).symm
   | n + 1 => by
     rw [Function.iterate_succ', Int.ofNat_succ, ← sub_sub]
-    exact congr_arg _ (pred_iterate n)
+    exact congr_arg _ (pred_iterate a n)
 #align int.pred_iterate Int.pred_iterate
 
 instance : IsSuccArchimedean ℤ :=
-  ⟨fun a b h =>
+  ⟨fun {a b} h =>
     ⟨(b - a).toNat, by
-      rw [succ_eq_succ, succ_iterate, to_nat_sub_of_le h, ← add_sub_assoc, add_sub_cancel']⟩⟩
+      rw [succ_eq_succ, succ_iterate, toNat_sub_of_le h, ← add_sub_assoc, add_sub_cancel']⟩⟩
```

Comment:
Could you please add an `#align` statement for `to_nat_sub_of_le`? Probably somewhere in `Init.Data.Int`.

Reply:
I checked this and there already seems to be an align in `Mathlib/Data/Int/Order/Basic.lean`